src/renderer/components/uielements/assets/assetIcon/AssetIcon.tsx

```diff
@@ -16,9 +16,7 @@ interface Props extends React.HTMLAttributes<HTMLDivElement> {
   asset: Asset
 }
 
-const AssetIcon: React.FC<Props> = (props: Props): JSX.Element => {
-  const { asset, size = 'normal' } = props
-
+const AssetIcon: React.FC<Props> = ({ asset, size = 'normal', ...rest }): JSX.Element => {
```

Comment:
I would always try to avoid `...rest` - it's just unsafe.

Reply:
yep but..... this is a props interface :sweat_smile:
```ts
interface Props extends React.HTMLAttributes<HTMLDivElement> {
  size?: Size
  asset: Asset
}
```
to tell the truth i would not like to pass all HTMLAtrributes :)

Reply:
This is an artifact migrated from BEPSwap. We don't need to pass all `HTMLAttributes` into `AssetIcon`, maybe none of then.